pkg/lib/audit/log.go

```diff
@@ -0,0 +1,48 @@
+package audit
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/authgear/authgear-server/pkg/api/event"
+)
+
+type Log struct {
+	ID           string                 `json:"id"`
+	CreatedAt    time.Time              `json:"createdAt"`
+	ActivityType string                 `json:"activityType"`
```

Comment:
I think we use underscore rather than camelcase for JSON key? Or remove the annotation first if we don't need it in this stage?

Reply:
Add a commit to explain the reason for using camel case in JSON key.